syft/core/frameworks/torch/utils.py

```diff
@@ -36,7 +38,12 @@ def get_child_command(obj, child_types=[]):
     """
     # Torch tensor or variable, or sy._SyftTensor
     if (is_tensor(obj) or is_variable(obj) or is_syft_tensor(obj)) and not isinstance(obj, str):
-        return obj.child, [type(obj.child)]
+        obj_type = type(obj.child)
+        # We identify Parameter type with Variable type since they are quite close
+        # (See test_operation_with_variable_and_parameter)
+        if obj_type is sy.Parameter:
+            obj_type = sy.Variable
```

Comment:
are we sure this is a good idea? Parameter is something that allows PyTorch to automatically pick up params when you call .get_parameters()...which all the optimizers do. It's possible that converting Parameter objects to Variable objects would cause them to get skipped by the optimizers.